### PR TITLE
DEV: Add rubocop-discourse gem

### PR DESCRIPTION
**Pull request by CvX**
Supersedes/closes #6

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
```
Comment (tshenry) [inline Gemfile:5]: I'm not sure if it matters either way, but do we need this? I don't see it in any of our other plugins.

**Review by tshenry — APPROVED** [review]
While rubocop stuff is fairly new to me, it looks like this follows the current way we do things when looking at https://github.com/discourse/rubocop-discourse/blob/master/README.md and other plugin repos 🙂

**Review by SamSaffron — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (SamSaffron) [reply Gemfile:5]: yeah looks a bit odd.

Comment (CvX) [reply Gemfile:5]: That's the current template from `bundle init`. I'm removing the line.

**Pull request merged**